### PR TITLE
Add database schema instructions

### LINKED ISSUE #3480: Re-add a second `launch` method without alternateComposedTaskRunnerApp parameter
In `TaskOperations` we currently have:

```java
long launch(String name, Map<String, String> properties, List<String> arguments, String alternateComposedTaskRunnerApp);
```

This is a recent change and technically a breaking change of the Rest Client. We may want to consider re-adding a second `launch` method without the `alternateComposedTaskRunnerApp` parameter to remove a breaking change.

### PR DESCRIPTION
**Pull request by jvalkeal**
- Fixes #3480

### CODE REVIEW

File: src/db/db2/V1-dataflow.sql [hunk 1]
```diff
@@ -0,0 +1,165 @@
+create sequence hibernate_sequence start with 1 increment by 1;
+
+create table app_registration (
```
Comment (ilayaperumalg) [inline src/db/db2/V1-dataflow.sql:3]: Do you think we should switch to a specific case (probably upper case) in these DDL scripts? I remember we had some case sensitive issues in the past.

Comment (jvalkeal) [reply src/db/db2/V1-dataflow.sql:3]: We can't as some db's will break. In fact it was task/batch which never should've use uppercase.

**Comment by ilayaperumalg** [comment]
@jvalkeal Do we want to confirm the schema changes by running database-specific tests with these changes before merging this PR?

**Comment by jvalkeal** [comment]
We don't have facilities(prob gonna look into it) to execute tests with these schema files and even if we had, we'd first need to merge these.

**Comment by ilayaperumalg** [comment]
Merging the changes into 2.6.x development line as we'll explore ways to test multiple databases.

**Pull request merged**